File: UserLogin.cs
// Jorge Calzada

/*
    Check the access of a user (username and password) and if wrong 3 times deny
    entry and terminate
*/

using System;

public class UserLogin
{
    public static void Main()
    {
        int times = 0;
        string user, password;
        const string correctUser = "jorge", correctPassword = "1234";

        do
        {
            if (times != 0)
            {
                Console.WriteLine("Access denied");
                Console.WriteLine();
            }

            Console.Write("Enter the user: ");
            user = Console.ReadLine();
            user = user.ToLower();

            Console.Write("Enter the password: ");
            password = Console.ReadLine();

            times++;
        }
        while ((user != correctUser || password != correctPassword) && times < 3);


        if (user == correctUser && password == correctPassword)
        {
            Console.WriteLine("Access granted");
        }
    }
}

File: UserLogin.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int times = 0;
    string user, password;
    const string correctUser = "jorge", correctPassword = "1234";
    locale loc;

    do
    {
        if (times != 0)
        {
            cout << "Access denied" << endl << endl;
        }

        cout << "Enter the user: ";
        cin >> user;
        transform(user.begin(), user.end(), user.begin(), ::tolower);

        cout << "Enter the password: ";
        cin >> password;

        times++;
    }
    while ((user != correctUser || password != correctPassword) && times < 3);


    if (user == correctUser && password == correctPassword)
    {
        cout << "Access granted";
    }

	return 0;
}

